components/camel-dropbox/pom.xml

```diff
@@ -107,6 +107,18 @@
                 <artifactId>camel-maven-plugin</artifactId>
                 <version>${project.version}</version>
             </plugin>
+            <plugin>
```

Comment:
Without it "mvn install" from within the component module doesn't create the sources jar (perhaps I'm doing it wrong?)